lenskit/algorithms/basic.py

```diff
@@ -153,6 +153,49 @@ def inverse_transform(self, ratings):
             rvps['rating'] += rvps['u_off'].fillna(0)
         return rvps.drop(columns=['u_off', 'i_off'])
 
+    def transform_user(self, user, ratings):
+        """
+        Transform a user's ratings by subtracting the bias model.
+
+        Args:
+            ratings(pandas.Series): The user's ratings, indexed by item.
+                Must have at least `item` as index and `rating` column.
+
+        Returns:
+            pandas.Series:
+                The transformed ratings and the user bias.
+        """
+        ratings_df = pd.DataFrame({
+            'rating': ratings.tolist(),
+            'item': ratings.index.tolist(),
+            'user': user})
+        return self.transform(ratings_df)
```

Comment:
A few things we need to change here:

- It should return a series, not a data frame
- It should compute the user's offset from *this* set of ratings
- It should return both the ratings and the user bias: `return ratings, u_bias` (this is so we have it available in to pass to `inverse_transform_user`).

Reply:
If local user_offsets is calculated in transform_user and the one created in fit is not used, then, what should I use  in inverse_transform_user method when user_bias=None? The self.user_offsets_ or calculate a new one?

Reply:
There is no basis for calculating a new one. Use `self.user_offsets_` if available, otherwise use 0.